bin/update_cucumber

```diff
@@ -12,6 +12,12 @@
 ini_set('error_reporting', E_ALL);
 set_error_handler(static fn ($code, $msg, $file, $line) => throw new ErrorException($msg, $code, $code, $file, $line));
 
+if (ini_get('zend.assertions') !== '1') {
+    // Ensure our assertions are actually processed. Otherwise any code inside the assert() call is compiled out
+    echo "This script must be run with zend.assertions set to 1\n";
+    exit;
```

Comment:
this should exit with a failure exit code

Reply:
🤦‍♂️indeed it should. Will fix, thanks